### PR TITLE
RemoveIncompleteUpload with Args.

### PR DESCRIPTION
**Pull request by BigUstad**
Functional tests, doc changed.

### CODE REVIEW

**Comment by Praveenrajmani** [comment]
Can you fix the conflicts @BigUstad

**Comment by BigUstad** [comment]
> Can you fix the conflicts @BigUstad

Conflicts resolved. Please check @Praveenrajmani @sinhaashish

File: Minio/ApiEndpoints/ObjectOperations.cs [hunk 1]
```diff
@@ -118,20 +123,73 @@ public IObservable<Upload> ListIncompleteUploads(ListIncompleteUploadsArgs args,
...
-            RestRequest request = await this.CreateRequest(args).ConfigureAwait(false);
-            response = await this.ExecuteAsync(this.NoErrorHandlers, request, cancellationToken).ConfigureAwait(false);
+            try
+            {
+                RestRequest request = await this.CreateRequest(args).ConfigureAwait(false);
+                response = await this.ExecuteAsync(this.NoErrorHandlers, request, cancellationToken).ConfigureAwait(false);
+            }
+            catch (Exception ex)
+            {
+                if (ex.GetType() == typeof(BucketNotFoundException))
```
Comment (Praveenrajmani) [inline Minio/ApiEndpoints/ObjectOperations.cs:141]: Why are we ignoring this error here?

Comment (BigUstad) [reply Minio/ApiEndpoints/ObjectOperations.cs:141]: This is an error encountered when procuring the list of incomplete uploads. If the bucket has been removed, the uploads have been removed as well, along with it. I encountered it while running the tests.

**Review by Praveenrajmani — APPROVED** [review]

**Review by sinhaashish — APPROVED** [review]

**Pull request merged**